internal/vxproto/module.go

```diff
@@ -420,23 +434,39 @@ func (ms *moduleSocket) sendPacket(ctx context.Context, dst string, pType Packet
 		Payload: payload,
 	}
 	if ms.IProtoIO == nil {
-		return fmt.Errorf("the ProtoIO is nil")
+		return ErrProtoIOUnset
 	}
-	if ms.HasIMCTokenFormat(dst) {
-		idstms := ms.GetIMCModule(dst)
+	sendViaIMC := func(token string) error {
+		idstms := ms.GetIMCModule(token)
 		if idstms == nil {
-			return fmt.Errorf("destination is unreachable")
+			return ErrDstUnreachable
 		}
 		dstms, ok := idstms.(*moduleSocket)
 		if !ok {
-			return fmt.Errorf("destination is not a module socket")
+			return ErrDstMalformed
 		}
 		packet.Module = dstms.GetName()
 		packet.Src = ms.GetIMCToken()
 		packet.ackChan = make(chan struct{}, 1)
 		packet.ctx = ctx
 		return dstms.recvPacket(ctx, packet)
 	}
+	if ms.HasIMCTopicFormat(dst) {
+		iti := ms.GetIMCTopic(dst)
```

Comment:
Maybe it is better to use more descriptive name for a variables, something like "topic" rather then "iti"?

Reply:
it's local variable which is using two times not below than 5 rows from declaration. I don't know why there need to use long variable names in such cases.